scripts/viz/tsne_viz.R

```diff
@@ -1,25 +1,47 @@
 # Pan-Cancer Variational Autoencoder
 # Gregory Way 2017
 #
-# Visualize t-SNE plots of VAE and raw RNAseq data features
+# Visualize t-SNE plots of VAE, ADAGE, and raw RNAseq data features
 
 library(dplyr)
 library(ggplot2)
 
```

Comment:
One last comment: Think you need to add library(readr) here

Reply:
In R, the [`double colon notation`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/ns-dblcolon.html) extracts the function from the namespace directly, so there is no need to load the library in this case.

Reply:
I had issues running the script without loading the package first.  Maybes its a versioning issue, which in that case your env script will address

Reply:
hmm, interesting. I will cross reference this note in the issue

Reply:
I also just added loading the library in caaa13f so that others don't run into this issue before I address #115